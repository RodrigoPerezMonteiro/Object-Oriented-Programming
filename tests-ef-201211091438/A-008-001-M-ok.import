VEGET|James T. Kirk|kirk@starfleet.net
OMNIV|Darth Vader|vader@imperial.net
VEGET|Master Yoda|yoda@jedi.edu
VEGET|James T. Kirk Jr.|kirkJr@starfleet.net
VEGET|Professor Yoda|yodaPhD@jedi.edu
VEGET|Relvinhas Esperto|only4@port.pt
OMNIV|Relvado Sintetico|relvas@gov.pt
OMNIV|Relv Apupado da Silva|mentesempre@gov.pt
VEGET|Rel Sabem O Resto|semEmail@gov.pt
