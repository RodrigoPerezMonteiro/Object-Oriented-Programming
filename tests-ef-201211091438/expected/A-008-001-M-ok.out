Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: VEGETARIAN|James T. Kirk|kirk@starfleet.net|0|0|0
VEGETARIAN|James T. Kirk Jr.|kirkJr@starfleet.net|0|0|0
OMNIVOROUS|Relv Apupado da Silva|mentesempre@gov.pt|0|0|0
VEGETARIAN|Relvinhas Esperto|only4@port.pt|0|0|0
OMNIVOROUS|Relvado Sintetico|relvas@gov.pt|0|0|0
VEGETARIAN|Rel Sabem O Resto|semEmail@gov.pt|0|0|0
OMNIVOROUS|Darth Vader|vader@imperial.net|0|0|0
VEGETARIAN|Master Yoda|yoda@jedi.edu|0|0|0
VEGETARIAN|Professor Yoda|yodaPhD@jedi.edu|0|0|0
Gestão de Clientes
1 - Visualizar clientes
2 - Registar cliente
3 - Alterar tipo
4 - Inscrever em lista
5 - Remover de lista
6 - Listar mensagens
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 